.github/workflows/ci.yml

```
@@ -107,3 +107,21 @@ jobs:
           python-version: ${{env.PYTHON_LATEST}}
       - run: python -m pip install -e .[dev]
       - run: python -c 'import attr; print(attr.__version__)'
+
+  check:  # This job does nothing and is only used for the branch protection
+    if: always()
+
+    needs:
+      - coverage
+      - install-dev
+      - package
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Decide whether the needed jobs succeeded or failed
+        uses: re-actors/alls-green@release/v1
```

In case, you'll want to pin it:
```suggestion
        uses: re-actors/alls-green@198badcb65a1a44528f27d5da555c4be9f12eac6  # v1.2.0
```